Makefile

```diff
@@ -2,7 +2,10 @@ pre-commit-run-all:
 	GOARCH=amd64 CGO_ENABLED=1 pre-commit run --all-files
 
 install-go-deps:
-	echo "no deps"
+	script/install-go-deps.sh
 
-run-e2e-tests:
-	echo "no e2e tests!"
+test:
+	go test ./... --count=1
```

Comment:
what does count guarantee here?

Reply:
The --count flag is an indirect way to disable cached test results, so when used the tests will always run.

Reply:
OK I will also use this in the adapter unit tests. Resolving.